diff --git a/requirements-build.txt b/requirements-build.txt
--- a/requirements-build.txt
+++ b/requirements-build.txt
@@ -1,6 +1,5 @@
 # Should be mirrored in pyproject.toml
 build
-cmake>=3.26
 packaging
 setuptools>=61
 torch
diff --git a/install_ascend.sh b/install_ascend.sh
--- a/install_ascend.sh
+++ b/install_ascend.sh
@@ -140,8 +140,8 @@ echo "Building TileLang with make..."
 # Other wise, make will use all available cores
 # and it may cause the system to be unresponsive
 CORES=$(nproc)
 MAKE_JOBS=$(( CORES * 50 / 100 ))
-make -j${MAKE_JOBS}
+make -j
 
 if [ $? -ne 0 ]; then
     echo "Error: TileLang build failed."
